middleware-jackson/src/test/java/com/nytimes/android/external/store3/middleware/jackson/JacksonReaderParserStoreTest.kt

```diff
@@ -0,0 +1,125 @@
+package com.nytimes.android.external.store3.middleware.jackson
+
+
+import com.fasterxml.jackson.core.JsonFactory
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.nytimes.android.external.store3.base.Fetcher
+import com.nytimes.android.external.store3.base.Persister
+import com.nytimes.android.external.store3.base.impl.BarCode
+import com.nytimes.android.external.store3.base.impl.StoreBuilder
+import io.reactivex.Maybe
+import io.reactivex.Single
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.mockito.Mock
+import org.mockito.Mockito.*
+import org.mockito.MockitoAnnotations
+import java.io.Reader
+import java.io.StringReader
+
+class JacksonReaderParserStoreTest {
+    @Rule
+    @JvmField
+    var expectedException = ExpectedException.none()
+    @Mock
+    internal var fetcher: Fetcher<Reader, BarCode>? = null
```

Comment:
Nit: `internal` is probably a Java->Kotlin conversion, can be `private` I think.